cli/go.mod

```diff
@@ -96,8 +96,8 @@ require (
 	github.com/gorilla/websocket v1.5.0 // indirect
 	github.com/gosuri/uitable v0.0.4 // indirect
 	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
-	github.com/hashicorp/consul/api v1.22.0-rc1 // indirect
-	github.com/hashicorp/consul/envoyextensions v0.3.0-rc1 // indirect
+	github.com/hashicorp/consul/api v1.28.4 // indirect
```

Comment:
Ah forgot to write a comment on this one-- I tried to get it to match 1.27.2 but the matching version of troubleshoot for this release line kept pulling in 1.28.4, even if I removed api changes and tried to specifically pull in 1.27.2. I figured it was ok because the CLI should not be used as a library

Reply:
Fair and probably means we screwed up the submodule deps graph. SGTM!